content/en/blog/falcosidekick-response-engine-part-8-fluxv2.md

```diff
@@ -0,0 +1,243 @@
+
+As you can see in the above arguments, we should pass `GITHUB_TOKEN` as an argument to our CLI application, in order to do that in Kubernetes environment, the best option is storing `GITHUB_TOKEN` in a Kubernetes Secret and [using Secret as environment variables](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables).
+
+```shell
+kubectl create namespace falco
+kubectl create secret generic github-secret \
+         --from-literal GITHUB_TOKEN=$GITHUB_TOKEN \
+         --namespace falco
+```
+
+Last but least, we're going to use `ko` tool which is created by Google to build, push our container image and deploy the project on Kubernetes cluster. `ko` is a simple, fast container image builder for Go applications.
+
+If you want to learn more about it, you can follow the [official repository](https://github.com/google/ko) of the project.
+
+We are also going to use `Container Registry` as an image repository service provided by the Google Cloud to Store, manage, and secure your Docker container images, but this is not an only the option for this demo, you can use `DockerHub`, `quay.io`, etc.
+
+```shell
+export PROJECT_ID=$(gcloud config get-value project)
+export KO_DOCKER_REPO=gcr.io/$PROJECT_ID
+ko apply -f falcoeventlistener.yaml
+```
+
+> If you are using private container registry, don't forget to create a registry secret to be able to pull and push images. You can follow the following guide to achieve this:                                                       
+https://colinwilson.uk/2020/07/09/using-google-container-registry-with-kubernetes/
+
+If everything works well, we should see the similar output below:
+
+```shell
+$ kubectl get pods --namespace falco
+NAME                   READY   STATUS    RESTARTS   AGE
+falco-event-listener   1/1     Running   0          59s
+```
+
+The last thing we have to do is installing `Falco`, `Falcosidekick` and configuring `Falcodekick` to forward events to our project.
+
+### Installing Falco and Falcosidekick
+
+There is a up-to-date and detail guide available to install `Falco` and `Falcosidekick`, you can follow this [link](https://github.com/falcosecurity/charts/blob/master/falcosidekick/README.md#installing-the-chart) to learn more about installing them.
+
+Here is that we are going to enable `CloudEvents` support of Falcosidekick.
+
+```shell
+helm repo add falcosecurity https://falcosecurity.github.io/charts
+helm repo update
+helm upgrade --install falco falcosecurity/falco \
+  --namespace falco \
+  --set falcosidekick.enabled=true \
+  --set falcosidekick.webui.enabled=true \
+  --set falcosidekick.config.cloudevents.address=http://falco-event-listener
+```
+
+Verify if everything is working before move on to the next step.
```

Comment:
Verify that everything is working as expected:

Reply:
thank you so much for fixing the issues, done.